base/db/R/query.prior.R

```diff
@@ -31,7 +32,7 @@
 ##'   con <- db.open(...)
 ##'   query.priors("ebifarm.pavi", c("SLA", "Vcmax", "leaf_width"), con = con)
 ##' }
-query.priors <- function(pft, trstr = NULL, con = NULL, ...){
+query.priors <- function(pft, trstr = NULL, con = NULL, ..., update){
```

Comment:
put update before the `...` that should be the final argument (list of leftover parameters)

Reply:
resolved by removing update as an argument in https://github.com/PecanProject/pecan/pull/2971/files/a2dc5df89b728d0b37a170d4434e445a9040b597#r928076659